### ISSUE TITLE
Plugin should allow using the FIPS AWS endpoint for FedRAMP

### ISSUE THREAD

**Issue by eyalmor94** [spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker#80]
For FedRAMP compliance with AWS, it's required to use the FIPS endpoints provided by AWS to create the lambda resources.

Currently there is no way to deploy a FedRAMP compliant AWS Lambda.

**Comment by nimakaviani** [comment]
I don't think it is a plugin issue. This appears to require a fix in how the Lambda driver is implemented in clouddriver.

**Comment by gsapkal** [comment]
This is not lambda specific . The clouddriver needs a way to configure AWS API to use FIPS endpoints for all communication in case of FedRAMP compliant deployments .

**Comment by nimakaviani** [comment]
agreed. I think we should raise this as an issue in clouddriver repo.
